public/media_engagement/worldbank_plr_sms_traffic.js

```diff
@@ -45,10 +45,18 @@ firebase.auth().onAuthStateChanged(function (user) {
 
 const TIMEFRAME_WEEK = 7;
 const TIMEFRAME_MONTH = 30;
-var chartTimeUnit = "1day";
+const EXTEND_X_AXIS_BY_HOURS = 32
```

Comment:
I also wonder if a fixed constant is the most appropriate solution here. It has the side effect of extending the x-axis beyond the current day when in the 1 day view. Wouldn't it be better to extend the x-axis to the end of the current day?

Reply:
I used 32hours because it resolved the problem when I was working on it but I agree working with 24 because I also think that the data from operators also affect the width of the bars,  therefore the margin of the error was large before since I had not resolved the list of operators.

Reply:
> I also wonder if a fixed constant is the most appropriate solution here. It has the side effect of extending the x-axis beyond the current day when in the 1 day view. Wouldn't it be better to extend the x-axis to the end of the current day?

By adding 24hours it extends the x-axis to the end of the current day since it accommodates the width of the bars